bind.go

```diff
@@ -90,6 +90,8 @@ func (b *DefaultBinder) BindBody(c Context, i interface{}) (err error) {
 		if err = b.bindData(i, params, "form"); err != nil {
 			return NewHTTPError(http.StatusBadRequest, err.Error()).SetInternal(err)
 		}
+	case ctype == "":
+		return ErrEmptyContentType
```

Comment:
I do not think adding new error variable that will eventually fall into `500` makes sense.  I think `
		return ErrUnsupportedMediaType.WithInternal(errors.New("missing content type header"))` would be better.

This is either error code 415 (StatusUnsupportedMediaType) or 400 (StatusBadRequest) as it is clients problem  but not 500 (StatusInternalServerError)